core/templates/domain/exploration/InteractionCustomizationArgObjectFactory.ts

```diff
@@ -0,0 +1,135 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
Please rename this to `interaction-customization-arg-object.factory.ts `.

I believe that is the new convention we are following.

/cc @seanlip @vojtechjelinek

Reply:
Done